companion/src/storage/storage.h

```diff
@@ -98,9 +98,14 @@ class StorageFormat
       return getFourCC(board);
     }
 
-    virtual bool isBoardCompatible(Board::Type board)
+    static bool isBoardCompatible(Board::Type board1, Board::Type board2)
```

Comment:
@bsongis Should we move this to `Boards` class?  With the getFourCC()?  `Boards` already manages getEEPromSize() and getFlashSize().  If anything, seems like all those go together.

Reply:
yes it makes sense